server/api/main.py

```diff
@@ -238,6 +238,10 @@ async def move_api_to_online():
             await _start_periodic_logs_collection()
             await _start_periodic_stop_logs()
 
+    # periodic project summaries calculation should run on all instances
+    # not just the chief
+    _start_periodic_project_summaries_calculation()
```

Comment:
not sure I understand why run it on workers. that means each worker can hold a different calculation on only 1 of them will have the most up to date one. if the chief does this anyway, its not really offloading cpu usage from it.

Reply:
I think its debatable if its better to calculate on all APIs or to redirect to the chief if it's already cached it should be a quick response. I'm ok with this for now but we might want to stress test it